contents/tutorials/nextjs-analytics.md

```diff
@@ -1,51 +1,54 @@
 ---
 title: 'How to set up Next.js analytics, feature flags, and more'
-date: 2025-05-22
+date: 2025-08-13
 author:
   - ian-vanagas
 showTitle: true
 sidebar: Docs
-featuredTutorial: true
-featuredVideo: 'https://www.youtube-nocookie.com/embed/nSBjr1Sz18o'
 tags:
   - configuration
   - feature flags
   - persons
   - events
 ---
 
-Next.js is a popular web framework built on React. It provides optimizations and abstractions to help developers build fast and performant apps and sites.
+Next.js is one of the web's most popular frameworks. Built on React, it provides optimizations and abstractions to help developers build fast and performant apps and sites. 
 
-In this tutorial, we will:
+To make sure you get the most out of Next.js, you can use PostHog to track events, identify users, use feature flags, and more. To help you get started, this tutorial walks you through:
 
-1. Build a basic Next.js blog with user authentication
-2. Add PostHog to it
-3. Set up the features of PostHog like custom event capture, user identification, and feature flags
+1. Building a basic Next.js (pages router) blog with user authentication
+2. Adding PostHog to it
+3. Setting up the features of PostHog like custom event capture, user identification, and feature flags
 
-> If you use Next.js with the **app router**, check out our other [Next.js app router analytics tutorial](/tutorials/nextjs-app-directory-analytics).
+
+import { CalloutBox } from 'components/Docs/CalloutBox'
+
+<CalloutBox type="info" title="Looking for the app router?">
+
+  If you use Next.js with the **app router**, check out our other [Next.js app router analytics tutorial](/tutorials/nextjs-app-directory-analytics).
+    
+</CalloutBox>
 
 ## Creating our Next.js app
 
-First, [install Node](https://nodejs.dev/en/learn/how-to-install-nodejs/) (14.6.0 or newer) and then run:
+First, [install Node](https://nodejs.dev/en/learn/how-to-install-nodejs/) (18.18 or newer) and then run:
```

Comment:
should we just say node 20? node 18 is eol at this point and I've had more and more trouble with it lately

Reply:
This is what Vercel says 🤷